components/MentorshipsList/MentorshipCard.js

```diff
@@ -34,59 +22,54 @@ const MentorshipCard = ({ mentorship }) => {
       <Stack
         direction={{ base: "column", lg: "row" }}
         spacing={{ base: 4, lg: 20 }}
+        width='100%'
       >
-        <Stack
-          pl={4}
-          direction={"column"}
-          textAlign={"center"}
-          justifyContent={"center"}
-        >
-          <Avatar
-            size="xl"
-            src={mentor.picture + "?tr=w-100,h-100,fo-auto"}
-            name={mentor.name}
-            alignSelf={"center"}
-          />
-          <Heading size="sm">{mentor.name}</Heading>
-          <Stack
-            direction={"row"}
-            textAlign={"center"}
-            justifyContent={"center"}
-          >
-            {mentor.twitter && (
-              <NextLink href={mentor.twitter} passHref>
-                <Link target="_blank">
-                  <Icon color="#1DA1F2" as={FaTwitterSquare} />
-                </Link>
-              </NextLink>
-            )}
-            {mentor.linkedin && (
-              <NextLink href={mentor.linkedin} passHref>
-                <Link target="_blank">
-                  <Icon color="#2867B2" as={FaLinkedin} />
-                </Link>
-              </NextLink>
-            )}
-          </Stack>
-        </Stack>
-
-        <Stack
-          direction={"column"}
-          textAlign={"left"}
-          justifyContent={"center"}
-        >
-          <ReactMarkdown components={ChakraUIRenderer(markdownTheme)} skipHtml>
-            {mentorship.message}
-          </ReactMarkdown>
-        </Stack>
-
-
+        <PersonInfo person={mentor} />
+        <MentorshipInfo mentorship={{ message, mentorshipDate, status, longterm }} />
         <Stack
           direction={"column"}
           textAlign={"right"}
           justifyContent={"center"}
         >
-          <Button colorScheme={"greenButton"}>Details</Button>
+          {
+          !!mentor?.mentor?.calendly
+          ?
+          <Button 
+            rightIcon={<ExternalLinkIcon />} colorScheme='greenButton' variant='solid'
+            onClick={() => window.open(mentor?.mentor?.calendly)}
+          >
+            Book a Call
+          </Button>
+          :
+          (mentor?.email &&
+          <Button
+          leftIcon={<EmailIcon />} colorScheme='greenButton' variant='solid'
+          onClick={() => window.open('mailto:'+mentor?.email)}
+          >
+            Email
+          </Button>)}
```

Comment:
```suggestion
                <Button
                  as="a"
                  leftIcon={<EmailIcon />} colorScheme='greenButton' variant='solid'
                  href={`mailto:${mentor.email}`}
                >)}
```
We can the use the `as` prop to get chakra-ui to render a link instead which is more accessible

Reply:
done